### ISSUE TITLE
läuft bei mir seit heute unter Chrome nicht

### ISSUE THREAD

**Issue by Topix1** [lostdesign/lss-layout-manager#18]
Bei mir funzt der LSS-Manager im Moment nicht. Nutze Chrome. Tampermonkey zeigt an, dass der LSS-Manager in der Version 2.61 mit Autoupdate aktiviert ist. Kriege ihn nicht zum Laufen. Oben im Bildschirm fehlt auch der Button zum Ein- und Ausschalten des LLS-Managers.

**Comment by Tsumiki-Chan** [comment]
Server laufen und bei mir läuft es mit Chrome und Tampermonkey.

Wie ist dein In-Game name?
Hast du das Skript doppelt aktiv?
Was zeigt dir die Konsole an, wenn du F12 drückst?

**Comment by Topix1** [comment]
Ingame Name ist >>>>>   Topix      (der war hier schon vergeben)

Console sagt >>>>>

Kann URL nicht laden: Die Domain dieser URL ist nicht den Domains der Applikation vorhanden. Um diese URL laden zu können, füge alle Domains und Subdomains deiner Applikation im Applikationsdomain-Feld in deinen Applikationseinstellungen hinzu.

---

.....und damit kann ich nicht wirklich was anfangen O_O

**Comment by Tsumiki-Chan** [comment]
Wenn du die URL
https://proxy.lss-manager.de/lss-layout-manager.user.js
aufrufst und sagst "Quelltext anzeigen", bekommst du diesen?

Der Fehlertext ist relativ nichtssagend für mich. Sieht für mich nicht einmal wie eine Chrome-Meldung aus q.q

**Comment by Topix1** [comment]
Bekomme eine Fehlermeldung vom Windows Script Host

Skript: C:\Users......
Zeile: 3
Zeichen: 1
Fehler: Objekt erwartet
Code: 800A138F
Quelle: Laufzeitfehler in Microsoft JScript

**Comment by Tsumiki-Chan** [comment]
Oke, das hilft mir schonmal ein kleines Stück weiter. Jetzt muss ich nur noch wissen, was für Microsoft die Zeile 3 ist

**Comment by Tsumiki-Chan** [comment]
@lostdesign Kannst du mal nachschauen was hier hinter steckt?

Zeile 3 müsste sein:
`$('head').append('<link id="lss_font_css" href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">').append('<link id="lss_icon_css" async rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">').append('<script id="lss_dashboard_js" async src="'+server+'/lss-dashboard.dev.js" type="text/javascript">').append('<script id="lss_chart_js" async src="'+server+'/highcarts.min.js" type="text/javascript">').append('<script async src="'+server+'/json2.js" type="text/javascript">').append("<style type='text/css' rel='stylesheet' id='switch-css'>.onoffswitch{position:relative;width:30px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none}.onoffswitch-checkbox{display:none}.onoffswitch-label{display:block;overflow:hidden;cursor:pointer;height:15px;padding:0;line-height:15px;border:2px solid #CCC;border-radius:15px;background-color:#FFF;transition:background-color .3s ease-in}.onoffswitch-label:before{content:'';display:block;width:15px;margin:0;background:#FFF;position:absolute;top:0;bottom:0;right:13px;border:2px solid #CCC;border-radius:15px;transition:all .3s ease-in 0}.onoffswitch-checkbox:checked + .onoffswitch-label{background-color:#49E845}.onoffswitch-checkbox:checked + .onoffswitch-label,.onoffswitch-checkbox:checked + .onoffswitch-label:before{border-color:#49E845}.onoffswitch-checkbox:checked + .onoffswitch-label:before{right:0}</style>");$("#news_li").before('<li>Redesign<div class="onoffswitch"><input class="onoffswitch-checkbox" id="s-redesign" name="onoffswitch" type="checkbox"><label class="onoffswitch-label" for="s-redesign"></label></div></li>');$("a[class^='navbar-brand hidden-xs']").append("<a href='http://forum.leitstellenspiel.de/index.php/Thread/8077-REDESIGNS-by-lost/'><img src='http://i.imgur.com/cPr2nKr.png' style='width:250px; top: 10px;left: 0; position: absolute !important;'></a>");$("a:contains('Sprechwunsch bearbeiten')").attr('id','sprechwunsch-bearbeiten');$("a:contains('Zurück zum Einsatz')").attr('id','zurueck-einsatz');$("a:contains('Anfahren')").addClass('naechstes-krankenhaus');$("a:contains('Rückalarmieren')").addClass('lf-zurueck');$("small:contains('Fahrzeuge ausgeblendet.')").css('display','none');$(".panel-heading:contains('Einsätze')").css('background-color','#e74c3c').css("color","#fff");$("div[class^='visible-xs']").before('<br>');$(".logo").css('display','none');if(typeof map=="undefined"){$('a[href$="backalarmAll"]').html("Eigene Fahrzeuge r&uuml;ckalarmieren");if($('#h2_vehicle_driving').length){$('a[href$="backalarmAll"]').parent().detach().appendTo('#h2_vehicle_driving');}else if($('#h2_vehicles_at_mission').length){$('a[href$="backalarmAll"]').parent().detach().appendTo('#h2_vehicles_at_mission');}`

**Comment by Topix1** [comment]
...jetzt komme ich schon deutlich an meine Grenzen. :-/
Aber von mir nochmal folgende Info >>>> Das einzige, was ich am PC gemacht hatte, war ein Java-Update, wenn ich mich recht entsinne. Chrome war zu der Zeit offen afaik. Da ich für eine Woche aber nicht zu Hause sein werde, komme ich derzeit nicht an meinen PC. 
Bin derzeit an einem anderen PC und dort habe ich keine Probleme mit dem LSS unter Chrome. 
Aber vielleicht wäre es grundsätzlich interessant, zu wissen, ob man es tunlichst vermeiden sollte, ein Java-Update bei geöffnetem LSS-Browser zu installieren. Vermutlich habe ich mir damit etwas zerschossen.

Würde es etwas bringen, wenn ich ggf. Chrome und/oder Java am anderen Rechner neu installiere?

**Comment by lostdesign** [comment]
Hey @Topix1, kann mir am Wochenende das zu Herzen nehmen. Java bringt nichts, weil Java nichts mit Javascript zu tun hat. 

Was mal interessant wäre, wenn du LSS am selben PC unter einem anderen Browser wie Firefox testest. Wenn das funktioniert, müsste es erstmal ein Chrome problem sein. Da wäre es hilfreich erstmal das Script & ggf. Tampermonkey runter zu hauen. 

Denke nicht das du Chrome neu installieren musst.

**Comment by Tsumiki-Chan** [comment]
@lostdesign Ich denke mal, dass da bei seinem Tampermonkey oder Chrome was verstellt ist. Bei mir läuft das Skript ja noch.

**Comment by Topix1** [comment]
Ich versuche das mal, wenn ich wieder am heimischen PC sitze. (Mitte nächster Woche)
Danke erstmal für die schnellen Antworten und das tolle Script. 
Wenn es plötzlich nicht mehr funzt, merkt man erst richtig, wie viel Unterstützung man dadurch erhält und wie sehr man sich daran gewöhnen kann!!

**Comment by Topix1** [comment]
So, Abschlußmeldung:

Unter Firefox installiert >>>>> klappt
unter Chrome das Script entfernt und neu installiert >>>> hat nicht gefunzt
unter Chrome Tampermonkey gelöscht und neu installiert, Script neu installiert >>>> klappt

Danke für eure Unterstützung!
